.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
```

Comment:
It's missing the preamble where you set the labels to show which board it flows into, and you can set a nice title  (see `Test plan: _______________`)

see other issue templates in both fleetdm/fleet and fleetdm/confidential repos for various examples